deps/v8/include/v8-version.h

```diff
@@ -11,7 +11,7 @@
 #define V8_MAJOR_VERSION 6
 #define V8_MINOR_VERSION 1
 #define V8_BUILD_NUMBER 534
-#define V8_PATCH_LEVEL 42
+#define V8_PATCH_LEVEL 43
```

Comment:
Now that V8 6.2 has landed, we can update the [embedder string](https://github.com/nodejs/node/blob/c39caa997c751473d0c8f50af8c6b14bcd389fa0/common.gypi#L30) instead.

Reply:
Thanks! Updated